pkgs/top-level/python-packages.nix

```
@@ -502,6 +502,8 @@ in {
 
   braintree = callPackage ../development/python-modules/braintree { };
 
+  breezy = callPackage ../development/python-modules/breezy { };
```

We need to add a symlink to make it compatible with bazaar:
```suggestion
  breezy = with python3Packages; (toPythonApplication breezy).overrideAttrs (self:
    { # symlink for bazaar compatibility
      postInstall = ''ln -s "$out/bin/brz" "$out/bin/bzr"'';
    });
```

---

I've moved this postInstall to the expression itself

---

Perfect.